operator/config/crd/bases/loki.grafana.com_lokistacks.yaml

```diff
@@ -5084,6 +5070,7 @@ spec:
                 - mode
                 type: object
             required:
+            - managementState
```

Comment:
I think we want to change `managementState` to be `optional` like it was always treated in the previous versions. There is a default for it, so the API _should_ fill in the value automatically.